code_examples/sdk_examples/src/core_features/claiming/05_verify_presentation.ts

```diff
@@ -2,8 +2,30 @@ import * as Kilt from '@kiltprotocol/sdk-js'
 
 export async function verifyPresentation(
   presentation: Kilt.ICredentialPresentation,
-  challenge: string | undefined = undefined
+  {
+    challenge,
+    trustedAttesterUris = []
+  }: {
+    challenge?: string
+    trustedAttesterUris?: Kilt.DidUri[]
+  } = {}
 ): Promise<void> {
   // Verify the presentation with the provided challenge.
   await Kilt.Credential.verifyPresentation(presentation, { challenge })
+
+  // Verify the credential attestation by checking on the blockchain.
+  const api = Kilt.ConfigService.get('api')
+  const attestationChain = await api.query.attestation.attestations(
+    presentation.rootHash
+  )
+  const attestation = Kilt.Attestation.fromChain(
+    attestationChain,
+    presentation.rootHash
+  )
+  if (attestation.revoked) {
+    throw new Error("Credential has been revoked and hence it's not valid.")
+  }
```

Comment:
unfortunately that is not all; we also need to check that the attestation matches the credential data (delegation & ctype). We got `verifyAgainstCredential` for that, although I'm seeing this seems broken to.... it doesn't check the delegation node id